eslint.config.mjs

```diff
@@ -1,47 +1,27 @@
 import unusedImports from 'eslint-plugin-unused-imports'
-import path from 'node:path'
-import { fileURLToPath } from 'node:url'
-import js from '@eslint/js'
-import { FlatCompat } from '@eslint/eslintrc'
+import eslint from '@eslint/js'
+import tseslint from 'typescript-eslint'
 
-const __filename = fileURLToPath(import.meta.url)
-const __dirname = path.dirname(__filename)
-const compat = new FlatCompat({
-    baseDirectory: __dirname,
-    recommendedConfig: js.configs.recommended,
-    allConfig: js.configs.all,
-})
-
-export default [
-    ...compat.extends('eslint:recommended'),
-    {
-        plugins: {
-            'unused-imports': unusedImports,
-        },
-
-        languageOptions: {
-            globals: {},
-        },
-
-        rules: {
-            'no-console': [
-                'error',
-                {
-                    allow: ['warn', 'error', 'log'],
-                },
-            ],
+export default tseslint.config(eslint.configs.recommended, tseslint.configs.recommended, {
+    plugins: {
+        'unused-imports': unusedImports,
+    },
+    languageOptions: {
+        globals: {},
+    },
 
-            'no-unused-vars': [
-                'error',
-                {
-                    ignoreRestSiblings: true,
-                    varsIgnorePattern: '_+',
-                    argsIgnorePattern: '^_',
-                },
-            ],
+    rules: {
+        '@typescript-eslint/no-unused-vars': [
+            'error',
+            {
+                ignoreRestSiblings: true,
+                varsIgnorePattern: '^_',
+                argsIgnorePattern: '^_',
+            },
+        ],
 
-            semi: ['error', 'never'],
-            'unused-imports/no-unused-imports': 'error',
-        },
+        semi: ['error', 'never'],
+        'unused-imports/no-unused-imports': 'error',
+        '@typescript-eslint/no-explicit-any': 'off',
```

Comment:
this feels like cheating but i also wanna do this in the TOA 🤣

Reply:
yah idk it seems like `any` is useful in [this](https://github.com/safeinsights/setup-app/blob/main/src/lib/types.ts#L29) line, but prob better to turn off linting for that line rather than the whole repo tbh

Reply:
Instead of turning off the check globally for the entire app, you can turn off rules line-by-line for special cases like this: https://github.com/safeinsights/management-app/blob/main/src/server/wrappers.ts#L7